masonry_testing/Cargo.toml

```diff
@@ -16,7 +16,7 @@ default-target = "x86_64-unknown-linux-gnu"
 targets = []
 
 [features]
-default = []
+default = ["masonry_core/default"]
```

Comment:
I don't think this feature edge is necessary, and I'd lean slightly towards not having it. Masonry Testing pretty intentionally doesn't re-export Masonry Core, so I don't think it should be in the business of enabling this feature.

That is, if you have a dependency on Masonry Testing (or Masonry Winit), to make it useful you by definition also have a dependency on Masonry or Masonry Core. That peer dependency should be the one enabling this kind of feature.

This isn't blocking.